### ISSUE TITLE
Feature Request: make mails more clearly arranged

### ISSUE THREAD

**Issue by Meeques** [wekan/wekan#5171]
Hi there!

Users are complaining about confusing / messy mails. 
Mails are sent as notification when something is changed on the board / on a card, etc.

These mails are not formatted in a clearly arranged manner, so users take the mails just as "something happend, I'll look at wekan" and then maybe it's something they didn't have to see on the board itself, but the mail would have been sufficient, if it was easy to overview. 

If you need a concrete proposal as to how these mails should look / what info they should hold, please let me know.

Thanks!

**Comment by xet7** [comment]
@Meeques 

> If you need a concrete proposal as to how these mails should look / what info they should hold, please let me know.

Yes, it would help to have concrete proposal about per-user and Admin Panel `Notifications Settings`. 

Some previous thoughts about combining emails etc are at:

https://github.com/wekan/wekan/issues/2026#issuecomment-940220596

There is link to Roadmap, but currently roadmap card opens to non-visible area, fix will come in new WeKan release in near future.

**Comment by Meeques** [comment]
Current format (2 notifications in one mail): "NAME created card "TEst" to list "To Do" at swimlane Social Network at board "Projekt Teilbereich C" LINK
[Projekt Teilbereich C] Excel Tabelle "Toollandkarte" ergänzen NAME commented on card "Excel Tabelle "Toollandkarte" ergänzen": "Test kommentar bitte ignorieren" at list "In Progress" at swimlane Social Network at board "Projekt Teilbereich C" LINK

Could be: 

-  **"[Projekt Teilbereich C](hyperlink to board)"**: **NAME** added a new card **"TEst"** to list **"To Do"** ~~at swimlane "Social Network" at~~

- **"[Projekt Teilbereich C](hyperlink to board)"**: **NAME** commented on card **"Toollandkarte ergänzen"**: "Test kommentar bitte ignorieren"
